stac_mjx/io.py

```diff
@@ -12,10 +12,94 @@
 import h5py
 from pathlib import Path
 from omegaconf import DictConfig
-import stac_mjx.io_dict_to_hdf5 as ioh5
-
-
-def load_data(cfg: DictConfig, base_path: Union[Path, None] = None):
+from omegaconf import OmegaConf
+from dataclasses import dataclass, asdict, field
+from typing import List, Dict
+
+
+# Dataclasses for config files and stac-mjx outputs
+
+
+@dataclass
+class ModelConfig:
+    """Configuration for body model."""
```

Comment:
Any chance we could document what the attributes mean in this and subsequent dataclass docstrings?